### PR TITLE
create task section

### LINKED ISSUE #6: Create Task section and style it
- task name
- task due date
- Each Task Same Class 
- Edit button 
- Delete button
- checkbox for done tasks
- icons from font awsome library

### PR DESCRIPTION
**Pull request by MohammedHasan-99**
consist of:
 - checkbox
 - task name
 - due date
 - remove botton
 - edit botton
 - 
 relates #6

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```
Comment (MMansour87) [inline .vscode/settings.json]: What this file for Bro?

**Review by MMansour87 — DISMISSED** [review]
Creative

**Review by Doaa-Abdulsalam — DISMISSED** [review]

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (sara219) [inline .vscode/settings.json]: delete `.vscode/settings.json`

**Review by sara219 — DISMISSED** [review]

**Review by FatmaHossam5 — APPROVED** [review]

**Review by sara219 — APPROVED** [review]

**Review by MMansour87 — APPROVED** [review]
Well Done

**Pull request merged**